app/assets/javascripts/views/data_portal/slides/ShareView.js

```diff
@@ -0,0 +1,67 @@
+(function (App) {
+  App.View.ShareView = App.Component.Slide.extend({
+
+    template: JST['templates/data_portal/slides/shareWidget'],
+
+    className: 'slide',
+
+    events: {
+      'click .js-back': '_onBack',
+      'click .js-copy': '_onCopy',
+      'blur  .js-copy': '_onBlur'
+    },
+
+    initialize: function (settings) {
+      this.constructor.__super__.initialize.call(this, settings);
+    },
+
+    _onBack: function () {
+      this.constructor.__super__.onCallback.call(this);
+    },
+
+    _onCopy: function (e) {
+      var copyButton = e.currentTarget;
+
+      window.getSelection().removeAllRanges();
+      var rangeSelection = document.createRange();
+      rangeSelection.selectNode(copyButton.previousElementSibling);
+      window.getSelection().addRange(rangeSelection)
+
+      var confirmation = document.execCommand('copy');
```

Comment:
They do: http://caniuse.com/#search=document.execCommand. I checked it before implement it.